classes/Tools.php

```diff
@@ -88,7 +93,7 @@ public static function resetRequest()
      *
      * @return bool|string Password
      */
-    public static function passwdGen($length = 8, $flag = 'ALPHANUMERIC')
+    public static function passwdGen($length = 8, $flag = null)
```

Comment:
Please update the PHPDoc :)

Reply:
```suggestion
    public static function passwdGen($length = 8, $flag = static::PASSWORDGEN_FLAG_ALPHANUMERIC)
```

For avoiding BC Breaks

Reply:
default case is alphanumeric, so no BC.

Reply:
It's a BC as you change the signature of the method.

Reply:
Ping @PululuK

Reply:
@Progi1984 can you give me an example of what could break something?

Reply:
A class which inherits of ToolsCore will be broken. Are you agree @PierreRambaud  ?

Reply:
Yes, if someone already overrides it, it will break the class.

Reply:
@PierreRambaud @Progi1984  something like http://sandbox.onlinephpfunctions.com/code/6c50a8a6e2a6ae0cb0caccc24327c35c016255d3 ?

Reply:
I forget that parameters are not typed -_-, so it doesn't break the code :sweat_smile: 
Afaik we use the Symfony Standard for Backward compatibility: https://symfony.com/doc/current/contributing/code/bc.html#id74

Regarding https://symfony.com/doc/current/contributing/code/bc.html#changing-classes change default argument value is not allowed and considered as a BC break
![image](https://user-images.githubusercontent.com/1462701/115031714-c3c44600-9ec8-11eb-9f5c-4d5c04ad9e20.png)

Reply:
@PierreRambaud but I don't add a value ... I simply change the value